apps/consulting/next.config.js

```diff
@@ -56,11 +56,16 @@ const nextConfig = {
         {
           // Proxying Plausible through Vercel:
           // https://plausible.io/docs/proxy/guides/vercel
-          source: '/plausible/js/script.js',
+
+          // `p7e` stands for Plausible. Why not just use `plausible`? The docs
+          // say: Choose a generic or irrelevant name for the subdirectory. If
+          // you choose something like analytics or plausible, it might get
+          // blocked in the future.
+          source: '/p7e/js/script.js',
```

Comment:
There's a good discussion about [ad blockers and proxying](https://plausible.io/docs/proxy/introduction) in the Plausible docs. 

I think I mostly agree with the Plausible point of view, although I think there is a risk here of disrespecting users who are not blocking for other reasons besides privacy. For example, perhaps they are attempting to limit all extraneous requests because they are working in a constrained environment, perhaps with limited bandwidth or power.

Reply:
Ah this answers my question about the API - 👍🏽